src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -3396,4 +3396,40 @@ void subscribeExclusiveTopicAndPattern(VertxTestContext context) throws Throwabl
         context.completeNow();
         assertTrue(context.awaitCompletion(TEST_TIMEOUT, TimeUnit.SECONDS));
     }
+
+    @Test
+    void openapiTest(VertxTestContext context) {
+        getRequest("/openapi")
+            .as(BodyCodec.jsonObject())
+            .send(ar -> {
+                context.verify(() -> {
+                    assertTrue(ar.succeeded());
+                    HttpResponse<JsonObject> response = ar.result();
+                    assertEquals(HttpResponseStatus.OK.code(), response.statusCode());
+                    JsonObject bridgeResponse = response.body();
+
+                    Map<String, Object> paths = bridgeResponse.getJsonObject("paths").getMap();
+                    // subscribe and unsubscribe are using the same endpoint but different methods
+                    int pathsSize = HttpOpenApiOperations.values().length - 1;
```

Comment:
so at this point, because you are leveraging on the OpenAPI operations, why don't do the check on the "operatioId" field?